scalableminds/webknossos
# Reduce the data that is being sent to the server on tree deletion

From the issue: "Tree deletion results in a ton of "deleteNode" and "deleteEdge" actions upon saving. We should compact these similar to tree merges / splits."
I added two tests for this, but would do some more manual testing before merging!

### Mailable description of changes (needs to be understandable by webknossos mailing list people):
- Fix performance for large tree deletion

### Steps to test:
- Delete trees and have a look in the network tab what is being sent to the server. There should only be a deleteTree update action but no deleteNode/deleteEdge update actions if the whole tree was deleted.
- Check whether the tracing is still functional after a reload.

### Issues:
- fixes #1907

------
